specification/labservices/resource-manager/readme.typescript.md

```diff
@@ -0,0 +1,12 @@
+## TypeScript
+
+These settings apply only when `--typescript` is specified on the command line.
+Please also specify `--typescript-sdks-folder=<path to root folder of your azure-sdk-for-js clone>`.
+
+``` yaml $(typescript)
+typescript:
+  azure-arm: true
+  package-name: "@azure/arm-LabServices"
+  output-folder: "$(typescript-sdks-folder)/packages/@azure/arm-LabServices"
```

Comment:
This repository uses the package name for its folder structure, so please change this too.
```suggestion
  output-folder: "$(typescript-sdks-folder)/packages/@azure/arm-labservices"
```